slides/intro-02-data-budget.qmd

```diff
@@ -23,85 +23,59 @@ knitr:
--   The city of Chicago releases anonymized trip-level data on taxi trips in the city.
--   We pulled a sample of 10,000 rides occurring in early 2022.
--   Type `?taxi` to learn more about this dataset, including references.
+-   The city of Long Beach releases data on animals at the Long Beach Animal Shelter.
+-   Our dataset is a sample of the cats entering and leaving the shelter.
+-   Type `?cat_adoption` to learn more about this dataset, including references.
 :::
 
 ::: {.column width="40%"}
-![](images/taxi_spinning.svg)
-:::
-
-:::
-
-::: footer
-Credit: <https://www.svgrepo.com/svg/8322/taxi>
-:::
-
-## Data on Chicago taxi trips
-
-::: columns
-::: {.column width="60%"}
--   `N = 10,000`
--   A nominal outcome, `tip`, with levels `"yes"` and `"no"`
--   Several **nominal** variables like pickup & dropoff location, taxi ID, and payment type.
--   Several **numeric** variables like trip length and fare subtotals.
+![](images/pet-house-blue.svg)
 :::
 
-::: {.column width="40%"}
-![](images/taxi.png)
-:::
 :::
 
 ::: footer
-Credit: <https://unsplash.com/photos/7_r85l4eht8>
+Credit: <https://www.svgrepo.com/svg/194185/pet-house-kennel>
 :::
 
-:::notes
-"Fare subtotals" refers to the fare itself, tax, tolls, tip amount. 
+## Data on shelter cats
 
-Also have date/time info
-:::
-
-## Checklist for predictors
-
-- Is it ethical to use this variable? (Or even legal?)
-
-- Will this variable be available at prediction time?
-
-- Does this variable contribute to explainability?
+```{r cats-quiet}
+#| echo: false
+data(cat_adoption, package = "modeldata")
+```
 
+-   N = `r nrow(cat_adoption)`
+-   A time-to-event outcome, consisting of the `time` spent at the shelter and the `event` status. 
+-   An event is a cat being homed by the Long Beach animal shelter. 
+-   If a cat is transfered to a different organization that works to home them, this is recorded as a non-event.
+-   Several **nominal** variables like sex, intake type and condition, as well as fur color.
+-   Two **numeric** variables for the location of the intake or capture, latitiude and logitude.
 
-## Data on Chicago taxi trips
+## Data on shelter cats
 
-```{r taxi-print}
+```{r cats-print}
 library(tidymodels)
+library(survival)
+library(modeldata)
 
-taxi
+cat_adoption
 ```
 
 :::notes
-`tip`: Whether the rider left a tip. A factor with levels "yes" and "no".
-
-`distance`: The trip distance, in odometer miles.
-
-`company`: The taxi company, as a factor. Companies that occurred few times were binned as "other".
-
-`local`: Whether the trip started in the same community area as it began. See the source data for community area values.
-
-`dow`: The day of the week in which the trip began, as a factor.
+FIXME: add decription of each variable?
+:::
 
-`month`: The month in which the trip began, as a factor.
+## Make the outcome
 
-`hour`: The hour of the day in which the trip began, as a numeric.
-:::
+```{r cats-surv-outcome}
+cat_adoption <- cat_adoption %>% 
```

Comment:
Dare we use the [survival hex logo](https://github.com/therneau/survival/blob/master/man/figures/logo.png) in the headers when we directly use those functions? 

Same for [ggsurvfit](https://github.com/pharmaverse/ggsurvfit/blob/main/man/figures/logo.png)?

Reply:
Yeah, that's a nice touch. I didn't so far because the hexes on the slides were meant to help navigation of the many tidymodels packages but I think extending it to survival-specific ones here is a good idea.